proposed/http-middleware/middleware.md

```diff
@@ -73,34 +72,34 @@ interface MiddlewareInterface
 {
     /**
      * Process an incoming server request and return a response, optionally delegating
-     * to the next middleware component to create the response.
+     * response creation to a handler.
      *
      * @param ServerRequestInterface $request
-     * @param DelegateInterface $delegate
+     * @param HandlerInterface $handler
      *
      * @return ResponseInterface
      */
     public function process(
         ServerRequestInterface $request,
-        DelegateInterface $delegate
+        HandlerInterface $handler
```

Comment:
Neither of those is necessarily accurate, as it depends entirely on context. Both `$delegate` and `$next` imply that the handler is not going to produce a response by itself, and that is an unknown.

Basically this boils down to "if the middleware cannot produce a response by itself, it will call the handler to produce one" and I can't think of any term besides `$handler` that is more clear on that point.